ci/cloudbuild/builds/clang-tidy.sh

```diff
@@ -45,9 +45,7 @@ io::run cmake "${cmake_args[@]}" \
 io::run cmake --build cmake-out
 
 mapfile -t ctest_args < <(ctest::common_args)
-io::run env -C cmake-out ctest "${ctest_args[@]}" -LE integration-test
-
-integration::ctest_with_emulators "cmake-out"
+io::run env -C cmake-out ctest "${ctest_args[@]}" 
```

Comment:
`-LE` **excludes** any test with a matching label (in this case `integration-test`) is the label. You need to revert the change to this line.

Reply:
Done